### PR TITLE
CAP-405

### PR DESCRIPTION
**Pull request by ZERO-94**

### CODE REVIEW

File: jobfair-frontend/src/components/customized-components/DecoratedBoothTool/DecoratedBoothSideBar/DecoratedBoothSideBar.component.jsx [hunk 1]
```diff
@@ -24,86 +27,186 @@ export const DecorateBoothSideBarComponent = (props) => {
...
...dv22uIEiLA4q7nvvCug8WTqzQveOH26fodo7g6uFe/a17W3+nFBAkRYENRdb1vkkz1CH9cPsVy/jrhr27PqMYvENYNlHAIesRiBYwRy0V+8iXP8+/fvX11Mr7L7ECueb/r48eMqm7FuI2BGWDEG8cm+7G3NEOfmdcTQw4h9/55lhm7DekRYKQPZF2ArbXTAyu4kDYB2YxUzwg0gi/41ztHnfQG26HbGel/crVrm7tNY+/1btkOEAZ2M05r4FB7r9GbAIdxaZYrHdOsgJ/wCEQY0J74TmOKnbxxT9n3FgGGWWsVdowHtjt9Nnvf7yQM2aZU/TIAIAxrw6dOnAWtZZcoEnBpNuTuObWMEiLAx1HY0ZQJEmHJ3HNvGCBBhY6jtaMoEiJB0Z29vL6ls58vxPcO8/zfrdo5qvKO+d3Fx8Wu8zf1dW4p/cPzLly/dtv9Ts/EbcvGAHhHyfBIhZ6NSiIBTo0LNNtScABFyNiqFCBChULMNNSdAhJyNSiECRCjUbEPNCRAhZ6NSiAARCjXbUHMCRMjZqBQiQIRCzTbUnAARcjYqhQgQoVCzDTUnQIScjUohAkQo1GxDzQkQIWejUogAEQo121BzAkTI2agUIkCEQs021JwAEXI2KoUIEKFQsw01J0CEnI1KIQJEKNRsQ80JECFno1KIABEKNdtQcwJEyNmoFCJAhELNNtScABFyNiqFCBChULMNNSdAhJyNSiECRCjUbEPNCRAhZ6NSiAARCjXbUHMCRMjZqBQiQIRCzTbUnAARcjYqhQgQoVCzDTUnQIScjUohAkQo1GxDzQkQIWejUogAEQo121BzAkTI2agUIkCEQs021JwAEXI2KoUIEKFQsw01J0CEnI1KIQJEKNRsQ80JECFno1KIABEKNdtQcwJEyNmoFCJAhELNNtScABFyNiqFCBChULMNNSdAhJyNSiECRCjUbEPNCRAhZ6NSiAARCjXbUHMCRMjZqBQiQIRCzTbUnAARcjYqhQgQoVCzDTUnQIScjUohAkQo1GxDzQkQIWejUogAEQo121BzAkTI2agUIkCEQs021JwAEXI2KoUIEKFQsw01J0CEnI1KIQJEKNRsQ80JECFno1KIABEKNdtQcwJEyNmoFCJAhELNNtScABFyNiqFCBChULMNNSdAhJyNSiEC/wGgKKC4YMA4TAAAAABJRU5ErkJggg=='
```
Comment (TienTruong2000) [inline jobfair-frontend/src/components/customized-components/DecoratedBoothTool/DecoratedBoothSideBar/DecoratedBoothSideBar.component.jsx:176]: please use image URL for this

Comment (TienTruong2000) [inline jobfair-frontend/src/components/customized-components/DecoratedBoothTool/DecoratedBoothSideBar/DecoratedBoothSideBar.component.jsx:176]: please use image URL for this

File: jobfair-frontend/src/containers/3D/JobFairBooth/AttendantJobFairBoothView.container.jsx [hunk 2]
```diff
@@ -3,10 +3,8 @@ import { BoothInfoMenuContainer } from '../../BoothInfoMenu/BoothInfoMenu.contai
 import { ControlTipsModalContainer } from '../../ControlTipModal/ControlTipsModal.container';
 import { InventoryContainer } from '../../Inventory/Inventory.container';
 import { JobFairBoothContainer } from './JobFairBooth.container';
-import ChatBoxContainer from '../../Agora/ChatBox/ChatBox.container';
-import { Card } from 'antd';
 import { useLocation } from 'react-router-dom';
-import ChatBox from '../../../components/Agora/ChatBox/ChatBox.component';
+import ChatBoxContainer from '../../Agora/ChatBox/ChatBox.container';
```
Comment (TienTruong2000) [inline jobfair-frontend/src/containers/3D/JobFairBooth/AttendantJobFairBoothView.container.jsx:7]: Why change?

Comment (TienTruong2000) [inline jobfair-frontend/src/containers/3D/JobFairBooth/AttendantJobFairBoothView.container.jsx:7]: Why change?

File: jobfair-frontend/src/containers/Agora/VideoCall/VideoCall.container.jsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ const VideoCallContainer = (props) => {
   const role = useSelector((state) => state.authentication?.user?.roles);
 
   async function initializeRTCClient(rtcClient, rtcToken, userId) {
-    rtcClient.on('user-joined', async (user, mediaType) => {
+    rtcClient.on('user-joined', async (user) => {
```
Comment (TienTruong2000) [inline jobfair-frontend/src/containers/Agora/VideoCall/VideoCall.container.jsx:22]: please retain the media type

Comment (TienTruong2000) [inline jobfair-frontend/src/containers/Agora/VideoCall/VideoCall.container.jsx:22]: please retain the media type

File: jobfair-frontend/src/containers/BoothInfoMenu/BoothInfoMenu.container.jsx [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
+import { Card, Tabs, Typography, notification } from 'antd';
 import { CompanyInformation } from '../../components/customized-components/BoothInfoMenu/BoothInformationTab/BoothInformationTab.component';
-import { Tabs, Typography, notification, Card } from 'antd';
 import { getCompanyBoothById } from '../../services/jobhub-api/CompanyBoothControllerService';
 import { getCompanyProfileAPI } from '../../services/jobhub-api/CompanyControllerService';
 import React, { useEffect, useState } from 'react';
 import SideBar from '../../components/commons/InfoMenu/InfoMenu.component';
 
 export const BoothInfoMenuContainer = (props) => {
-  const { companyBoothId, handleOpenDetail, isShow, activeKey, openInventory, chatBoxContainer } = props;
```
Comment (TienTruong2000) [inline jobfair-frontend/src/containers/BoothInfoMenu/BoothInfoMenu.container.jsx:11]: Why remove this?

Comment (TienTruong2000) [inline jobfair-frontend/src/containers/BoothInfoMenu/BoothInfoMenu.container.jsx:11]: Why remove this?

**Review by TienTruong2000 — CHANGES_REQUESTED** [review]

File: jobfair-frontend/src/containers/BoothInfoMenu/BoothInfoMenu.container.jsx (hunk 4 above)
Comment (ZERO-94) [reply jobfair-frontend/src/containers/BoothInfoMenu/BoothInfoMenu.container.jsx:11]: only delete isShow because unused var

**Review by TienTruong2000 — APPROVED** [review]
LGTM

**Pull request merged**